client/src/i18n/locales/pl.json

```diff
@@ -38,15 +38,17 @@
     "AboutMistakesJournal": {
       "HeaderFirstPart": "Wykorzystaj w pełni swoje błędy",
       "HeaderSecondPart": "prowadząc",
+      "EarlyAccessHeader": "Witamy we wczesnym dostępie!",
+      "EarlyAccessText": "Przekonaj się sam i daj znać do myślisz!",
       "Step1": {
         "StepName": "Rozpoznaj",
         "Header": "Błędy są cenne",
-        "Text": "Pomożemy Ci zapisywać oraz organizować Twoje błędy, tak aby czerpać z nich jak najwięcej."
+        "Text": "Pomożemy Ci zapisywać oraz organizować Twoje błędy, tak abyś mógł z nich czerpać jak najwięcej."
```

Comment:
I specially changed the text. Not every user is male :)

Reply:
Szymons request so i dont know what to do now